baybe/utils/dataframe.py

```diff
@@ -418,3 +418,24 @@ def fuzzy_row_match(
             inds_matched.extend(inds_found)
 
     return pd.Index(inds_matched)
+
+
+def pretty_print_df(df: pd.DataFrame) -> str:
+    """Convert a dataframe into a pretty/readable format.
+
+    This function returns a string representation containing at most the
+    first 10 rows and 7 columns of the dataframe.
+    In case the dataframe is empty, it returns a corresponding statement.
+
+    Args:
+        df: The dataframe to be printed.
+
+    Returns:
+        The values to be printed as a str table.
+    """
+    # Get first 10 rows and 7 columns and do not break the table in two lines
+    with pd.option_context(
+        "display.max_rows", 10, "display.max_columns", 7, "expand_frame_repr", False
```

Comment:
you still havent made `max_rows` and `max_columns` optional argumetns of this function

Id orefer default values so the dfs are even a bit smaller than now, ie `max_rows=6`, `max_columns=4`